components/brave_wallet_ui/common/constants/mocks.ts

```diff
@@ -366,7 +367,7 @@ export const mockSolDappSignAndSendTransactionRequest = {
     'eTldPlusOne': 'csb.app'
   },
   'groupId': undefined
-} as BraveWallet.TransactionInfo
+} as unknown as SerializableTransactionInfo
```

Comment:
this is disabling type checks for this object. Removing the `bigint` casts should resolve the type errors.

Reply:
```
export const mockSolDappSignAndSendTransactionRequest: SerializableTransactionInfo = {
  'id': 'e1eae32d-5bc2-40ac-85e5-2a4a5fbe8a5f',
  'fromAddress': mockSolanaAccount.address,
  'txHash': '',
  'txDataUnion': {
    'ethTxData': undefined,
    'ethTxData1559': undefined,
    'filTxData': undefined,
    'solanaTxData': {
      'recentBlockhash': 'C115cyMDVoGGYNd4r8vFy5qPJEUdoJQQCXMYYKQTQimn',
      'lastValidBlockHeight': '0',
      'feePayer': mockSolanaAccount.address,
      'toWalletAddress': '',
      'splTokenMintAddress': '',
      'lamports': '0',
      'amount': '0',
      'txType': 11,
      'instructions': [
        {
          'programId': '11111111111111111111111111111111',
          'accountMetas': [
            {
              'pubkey': mockSolanaAccount.address,
              'isSigner': true,
              'isWritable': true
            },
            {
              'pubkey': mockSolanaAccount.address,
              'isSigner': true,
              'isWritable': true
            }
          ],
          'data': [2, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 0],
          'decodedData': undefined
        }
      ],
      'sendOptions': undefined,
      'signTransactionParam': undefined
    }
  },
  'txStatus': 0,
  'txType': 11,
  'txParams': [],
  'txArgs': [],
  'createdTime': { 'microseconds': 1654540245386000 },
  'submittedTime': { 'microseconds': 0 },
  'confirmedTime': { 'microseconds': 0 },
  'originInfo': {
    'origin': {
      'scheme': 'https',
      'host': 'f40y4d.csb.app',
      'port': 443,
      'nonceIfOpaque': undefined
    },
    'originSpec': 'https://f40y4d.csb.app',
    'eTldPlusOne': 'csb.app'
  },
  'groupId': undefined
}
```